base_technical_user/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Base Technical User
+===================
+
+This module extends the functionality of company management.
+It allows you to bind a technical user on the company in order to use it in
+batch processes.
+
+The technical user must
+- be inactive to avoid login
```

Comment:
@guewen Interesting remark. I wasn't aware of that. Thanks. The other goal of making it inactive is to avoid Odoo taking care of this kind of users in the Enterprise contract.